docs/en/mutations.md

```diff
@@ -52,17 +52,17 @@ You can also dispatch mutations using objects:
 
 ``` js
 store.dispatch({
-  type: 'INCREMENT',
-  payload: 10
+  type: 'INCREMENT_BY',
+  payload: [ 10 ]
```

Comment:
@CthulhuDen I took a second look -- you're right.

> https://github.com/vuejs/vuex/blob/master/src/index.js#L83
> ```javascript
> payload = [type.payload]
> ```

This doesn't look super intentional.
I don't see a reason to spread an arguments array that will only ever have one element.